script.js

```diff
@@ -118,7 +190,33 @@ const amountConverter = (amount, rate) => {
 
 btn.addEventListener("click", () => {
   resultText.innerHTML =
-    amountConverter(inputAmount.value, objectRateSearcher()) +
+    amountConverter(
+      inputAmount.value,
+      objectRateFetcher(countriesToSelect.value)
+    ) +
     " " +
     countriesToSelect.value;
 });
+
+const addNewCurrency = (base, symbol, rate) => {
+  //Chekear si existe la moneda
+  for (let i = 0; i < currencyData.length; i++) {
+    if (currencyData[i].currency === currency) {
+      resultText.innerHTML = `The currency ${currency} already exists in our system.`;
+      return;
+    }
+  }
+
+  const timestamp = new Date().getTime();
+  // poner la moneda en el array
+  currencyData.push({ base, symbol, rate, timestamp });
+  resultText.innerHTML = `The currency ${currency} was added successfully to our system.`;
```

Comment:
Perfect, our new currency was added... but I don't see it in the list :(

How could we solve this? 🤔  _(hint: maybe a function that iterates through the list and adds new options to the DOM could help)_